frontend/src/AppRouter.tsx

```diff
@@ -208,6 +208,15 @@ const AppRouter: React.FC = () => {
           claim={[Claims.CONTACT_VIEW]}
           title={getTitle('View Contact')}
         />
+        <AppRoute
+          protected
+          exact
+          path="/research/list"
+          component={ResearchListView}
+          layout={AuthLayout}
+          claim={Claims.PROPERTY_VIEW}
```

Comment:
The backend controller requires LeaseView for the endpoints but the frontend is requiring PropertyView? I don't know story well enough but seems odd.